lametro/bills.py

```diff
@@ -319,7 +319,8 @@ def scrape(self, window=28, matter_ids=None) :
                          'CARRIED OVER' : 'deferral',
                          'RECEIVED' : 'receipt',
                          'REFERRED' : 'referral-committee',
-                         'FORWARDED DUE TO ABSENCES AND CONFLICTS' : 'committee-passage'}
+                         'FORWARDED DUE TO ABSENCES AND CONFLICTS' : 'committee-passage',
+                         'NO ACTION TAKEN': 'no-action-taken'}
```

Comment:
the possible values that can appear are part of controlled vocabulary: https://github.com/opencivicdata/python-opencivicdata/blob/dfbb926080b0698b4885cd5f1a08637117416021/opencivicdata/common.py#L113

I don't think it's worth adding a new type to that controlled vocabular, so I would set the value to `None`

Reply:
okey doke, reverted back to my original commit, which does exactly that.